packages/bundle/src/createCognitiveServicesSpeechServicesPonyfillFactory.js

```diff
@@ -12,6 +17,26 @@ export default function createCognitiveServicesSpeechServicesPonyfillFactory({
   subscriptionKey,
   textNormalization
 }) {
+  if (!credentials && (authorizationToken || region || subscriptionKey)) {
+    console.warn(
+      'botframework-webchat: "authorizationToken", "region", and "subscriptionKey" are being deprecated and will be removed on or after 2019-12-17. Please use "credentials" instead.'
+    );
+
+    credentials = async () => {
+      if (authorizationToken) {
```

Comment:
Doesn't this need to come from `credentials.authorizationToken`? I think I'm missing something.